File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.c
int main()
{
    int n, m;
    scanf("%d %d\n", &n, &m);

    int a[n];
    int i, j;
    int tmp;

    for(int i=0; i<n; i++)
        a[i] = i+1;

    for(int x=0; x<m; x++){
        scanf("%d %d\n", &i, &j);
        // 인덱스를 1씩 빼서 배열의 인덱스에 맞게 조정
        i--;
        j--;

        tmp = a[i];
        a[i] = a[j];
        a[j] = tmp;
    }

    for(int i=0; i<n; i++){
        printf("%d ", a[i]);
    }

    return 0;
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let [n, m] = input[0].split(" ").map(Number);
let arr = [];
for (let i = 1; i <= n; i++) {
  arr.push(i);
}

for (let i = 1; i <= m; i++) {
  let [a, b] = input[i].split(" ").map(Number);
  let tmp = arr[a - 1];
  arr[a - 1] = arr[b - 1];
  arr[b - 1] = tmp;
}

let answer = "";
for (let i = 0; i < n; i++) {
  answer += arr[i] + " ";
}
console.log(answer);